packages/@okta/vuepress-site/docs/guides/terraform-create-custom-auth-server/main/index.md

```diff
@@ -0,0 +1,407 @@
+
+1. Set `always_include_in_token` to `true` if you want to include the claims in the token. Otherwise, set it to `false`.
+
+For example:
+
+```hcl
+# access token
+resource "okta_auth_server_claim" "example" {
+  auth_server_id = okta_auth_server.api_server.id
+  name           = "staff"
+  value_type     = "EXPRESSION"
+  value          = "String.substringAfter(user.email, \"@\") ==
+                    \"mycompany.com\""
+  scopes         = [
+    okta_auth_server_scope.create_obj.name,
+    okta_auth_server_scope.read_obj.name,
+    okta_auth_server_scope.replace_obj.name,
+  ]
+  claim_type     = "RESOURCE"
+}
+
+# identity token
+resource "okta_auth_server_claim" "example2" {
+  auth_server_id = okta_auth_server.api_server.id
+  name           = "staff"
+  value_type     = "EXPRESSION"
+  value          = "String.substringAfter(user.email, \"@\") ==
+                    \"mycompany.com\""
+  claim_type     = "IDENTITY"
+}
+```
+
+### Add scopes
+
+Create one or more `okta_auth_server_scope` resources for the claims returned by the server.
+
+For each scope:
+
+1. Set `name` to the name of the OAuth scope.
+
+1. Set `auth_server_id` to the authorization server ID.
+
+1. Set `metadata_publish` to `NO_CLIENTS` to prevent publishing the OIDC scope metadata for clients for this scope to the Okta-hosted [OIDC metadata](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata) for this authorization server. To publish OIDC scope metadata, set `metadata_publish` to `ALL_CLIENTS`. The URL, the issuer URI with a special suffix, is unique for each authorization provider on the Okta server. If you publish the metadata, the authorization server includes it in the metadata generated for the URI.
+
+1. Set `consent` to `REQUIRED` to require the end user to approve granting the scope. Okta displays a consent dialog to allow the release of information. For example, when you sign in to a third-party app, Okta displays a consent prompt to enable the app access to details such as your email address. If the user chooses to opt-out, authorization fails. To skip the consent dialog, set `consent` to `IMPLICIT`.
+
+   Optionally, you can set `display_name` to the name of this scope that's displayed in the consent dialog. Otherwise the value of the `named` field is displayed.
+
+1. Optionally set `default` to `true` to support requests with empty scope lists from the requestor. This adds the scope to an access token when the client omits the scope parameter in a token request. It also requires an authorization server policy rule that permits the behavior.
+
+1. Set `optional` to `false` if this scope is required. Otherwise, set it to `false`. It's unsupported to set both optional and default to `true`, as they are mutually exclusive.
```

Comment:
I think it makes more sense to make a callout about optional and default being mutually exclusive.

```suggestion
1. Set `optional` to `false` if this scope is required. Otherwise, set it to `false`. 

> **Tip**: It's unsupported to set both `optional `and `default `to `true`, as they are mutually exclusive.
```

Reply:
Slight modification in the next commit as the second sentence on line 225 should be set it to `true`, not also to false ;-)